File: C#/Estruturas_de_controle.cs
//Estrutura if, else if e else
string caminhoEsquerdo = "obstaculo";

if (caminhoEsquerdo == "obstaculo")
{
    Console.WriteLine("Seguindo caminho da direita");
}
else if (caminhoEsquerdo == "ponteQuebrada")
{
    Console.WriteLine("Procurando outro caminho");
}
else
{
    Console.WriteLine("Seguindo caminho da esquerda");
}



bool possuiOvos = true;
string itensComprados = "";

if (possuiOvos)
{
    itensComprados = "Leite";
}
else
{
    Console.WriteLine("Passou na sessão de congelados");
    itensComprados = "Lasanha congelada";
}

Console.WriteLine("Item comprado: " + itensComprados);



int nivelDeFome = 1;

if (nivelDeFome == 1)
{
    Console.WriteLine("Pouca fome");
}
else if (nivelDeFome == 2)
{
    Console.WriteLine("Muita fome");
}
else
{
    Console.WriteLine("Você comeria mais que o Pica-Pau");
}



//Switch Case
string dia = "terça";

switch (dia)
{
    case "segunda":
        Console.WriteLine("Dia de começar a semana com energia!");
        break;

    case "terça":
        Console.WriteLine("Hora de aprender coisas novas!");
        break;

    case "quarta":
        Console.WriteLine("Metade da semana, continue firme!");
        break;

    case "quinta":
        Console.WriteLine("Quase lá!");
        break;

    case "sexta":
        Console.WriteLine("Fim de semana chegando!");
        break;

    default:
        Console.WriteLine("Final de semana!");
        break;
}



int[] numeros = {1,2,3,4,5};

for (int i = 0; i < numeros.Length; i++)
{
    Console.WriteLine(numeros[i]);
}



Random random = new Random();

int jogadas = 0;
bool cara = false;

while (!cara)
{
    jogadas++;
    cara = random.NextDouble() < 0.5;
}

Console.WriteLine($"A moeda caiu cara após {jogadas} jogadas!");



int numero;

do
{
    Console.Write("Digite um número entre 1 e 10: ");
    numero = int.Parse(Console.ReadLine());

} while (numero < 1 || numero > 10);

Console.WriteLine($"Você digitou o número {numero}");



for (int contador = 1; contador <= 5; contador++)
{
    Console.WriteLine(contador);
}



Random random = new Random();

int dado = 0;

while (dado <= 4)
{
    dado = random.Next(1, 7);
    Console.WriteLine("Você tirou um " + dado);
}



int bolinhosComidos = 0;

do
{
    bolinhosComidos++;
    Console.WriteLine($"Você comeu {bolinhosComidos} bolinhos!");

} while (bolinhosComidos < 5);

File: python/Estruturas_de_controle.py
#Estrutura if, else if e else
caminhoEsquerdo = "obstaculo"

if caminhoEsquerdo == "obstaculo":
    print("Seguindo caminho da direita")
elif caminhoEsquerdo == "ponteQuebrada":
    print("Procurando outro caminho")
else:
    print("Seguindo caminho da esquerda")


possuiOvos = True
itensComprados = ""

if possuiOvos:
    itensComprados = "Leite"
else:
    print("Passou na sessão de congelados")
    itensComprados = "Lasanha congelada"

print("Item comprado:", itensComprados)


nivelDeFome = 1

if nivelDeFome == 1:
    print("Pouca fome")
elif nivelDeFome == 2:
    print("Muita fome")
else:
    print("Você comeria mais que o Pica-Pau")



#Switch Case
dia = "terça"

match dia:
    case "segunda":
        print("Dia de começar a semana com energia!")
    case "terça":
        print("Hora de aprender coisas novas!")
    case "quarta":
        print("Metade da semana, continue firme!")
    case "quinta":
        print("Quase lá!")
    case "sexta":
        print("Fim de semana chegando!")
    case _:
        print("Final de semana!")


numeros = [1, 2, 3, 4, 5]

for numero in numeros:
    print(numero)


import random

jogadas = 0
cara = False

while not cara:
    jogadas += 1
    cara = random.random() < 0.5

print(f"A moeda caiu cara após {jogadas} jogadas!")


#Do While
while True:
    numero = int(input("Digite um número entre 1 e 10: "))

    if 1 <= numero <= 10:
        break

print(f"Você digitou o número {numero}.")


for contador in range(1, 6):
    print(contador)


import random

dado = 0

while dado <= 4:
    dado = random.randint(1, 6)
    print("Você tirou um", dado)


bolinhosComidos = 0

while True:
    bolinhosComidos += 1
    print(f"Você comeu {bolinhosComidos} bolinhos!")

    if bolinhosComidos >= 5:
        break